link/src/main/java/com/stripe/android/link/ComposeExtensions.kt

```diff
@@ -0,0 +1,26 @@
+package com.stripe.android.link
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalContext
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.stripe.android.link.injection.NativeLinkComponent
+import com.stripe.android.uicore.utils.extractActivity
+
+/**
+ * Retrieves or builds a ViewModel instance, providing the [NativeLinkComponent] (activity
+ * retained component) to the factory to facilitate its creation via dependency injection.
+ */
+@Composable
+internal inline fun <reified T : ViewModel> paneViewModel(
```

Comment:
ViewModel creation logic and general DI setup was inspired by [financial-connections](https://github.com/stripe/stripe-android/blob/060ad2edb0b100013f81feaf6a98447571bc7ac1/financial-connections/src/main/java/com/stripe/android/financialconnections/presentation/ComposeExtensions.kt)

Reply:
thanks! This is useful context. out of curiosity, do you know why this is called "pane" view model?

Reply:
And is this needed in this PR? It looks unused so far